components/micro-gateway-core/src/main/ballerina/src/gateway/analytics/faults_util.bal

```diff
@@ -21,19 +21,34 @@ public function getFaultMetaData(FaultDTO dto) returns string {
     return dto.metaClientType;
 }
 
-public function getFaultPayloadData(FaultDTO dto) returns string {
+public function getFaultPayloadData(FaultDTO dto, string amAnalyticsVersion) returns string {
+    printDebug(KEY_ANALYTICS_FILTER, "Generating fault data payload for " + amAnalyticsVersion);
+    string resourceTemplate = OBJ;
+    string applicationOwner = OBJ;
+    string properties = "";
+    // If am analytics version is 3.2.0, append api resource template and application owner
+    if (amAnalyticsVersion == DEFAULT_AM_ANALYTICS_VERSION) {
+        resourceTemplate = resourceTemplate + dto.apiResourceTemplate + OBJ;
+        applicationOwner = applicationOwner + dto.applicationOwner + OBJ;
+    }
+    // If analytics version is 3.1.0, append properties.
+    if (amAnalyticsVersion !== DEFAULT_AM_ANALYTICS_VERSION_300) {
```

Comment:
Do we need !== or !=

Reply:
It is !=.
Fixed in https://github.com/wso2/product-microgateway/pull/1319/commits/b694c6960318d34763d343da195320faf623a55f